controller/rest/model.go

```diff
@@ -138,6 +138,7 @@ func NewVolume(context *api.ApiContext, name string, readOnly bool, replicas int
 		v.Actions["shutdown"] = context.UrlBuilder.ActionLink(v.Resource, "shutdown")
 		v.Actions["snapshot"] = context.UrlBuilder.ActionLink(v.Resource, "snapshot")
 		v.Actions["revert"] = context.UrlBuilder.ActionLink(v.Resource, "revert")
+		v.Actions["deletevolume"] = context.UrlBuilder.ActionLink(v.Resource, "deletevolume")
```

Comment:
As its possible to have a volume with 0 replica's this should be delete replica's?

Reply:
We are deleting the whole volume and there is already deletereplica implemented which only remove the metadata of the replica not the contents in that directory. So i think deletevolume sounds good. What do you say ?